id_test.js

```diff
@@ -14,17 +14,17 @@ const semigroup = require('./laws/semigroup');
 const setoid = require('./laws/setoid');
 const traversable = require('./laws/traversable');
 
-const {tagged} = require('daggy');
 const {of, empty, concat, equals, map} = require('.');
 
 const Id = require('./id');
 
 // Special type of sum for the type of string.
-const Sum = tagged('v');
-Sum[of] = (x) => Sum(x);
-Sum[empty] = () => Sum('');
-Sum.prototype[of] = Sum[of];
-Sum.prototype[empty] = Sum[empty];
```

Comment:
At some point I thought `Sum('').constructor` did not evaluate to `Sum`. I was mistaken. I've reverted to the daggy version.